src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Email/EmailConfigurationType.php

```diff
@@ -100,6 +100,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('log_emails', SwitchType::class, [
                 'label' => $this->trans('Log Emails', 'Admin.Advparameters.Feature'),
             ])
-            ->add('smtp_config', SmtpConfigurationType::class);
+            ->add('dkim_enable', SwitchType::class, [
+                'attr' => ['class' => 'js-dkim-enable'],
+                'choices' => [
+                    'Disabled' => false,
+                    'Enabled' => true,
+                ],
+                'label' => $this->trans('DKIM signing', 'Admin.Advparameters.Feature'),
+                'help' => $this->trans('When enabling DKIM, fill the data below and properly test it afterwards. If no email is sent, check logs.', 'Admin.Advparameters.Help'),
```

Comment:
```suggestion
                'help' => $this->trans('Enable DKIM, fill in the fields and give it a try. If no email is sent, check logs.', 'Admin.Advparameters.Help'),
```
I adapted the wording here since the fields only appear when the feature is enabled (which is great! 😃)